build/templates/dotnet-install.yml

```diff
@@ -1,6 +1,21 @@
 ﻿parameters:
-  UnoCheck_Version: '1.27.4'
-  UnoCheck_Manifest: 'https://raw.githubusercontent.com/unoplatform/uno.check/870b2615fc2783e9355374c12015597a92dd242a/manifests/uno.ui.manifest.json'
+- name: UnoCheck_Version
+  type: string
+  default: '1.27.4'
+
+- name: UnoCheck_Manifest
+  type: string
+  default: 'https://raw.githubusercontent.com/unoplatform/uno.check/870b2615fc2783e9355374c12015597a92dd242a/manifests/uno.ui.manifest.json'
+
+# Whether the Android workload should be installed.
+- name: installAndroidWorkload
+  type: boolean
+  default: false
+
+# Whether the iOS workload should be installed.
+- name: installAppleWorkload
+  type: boolean
+  default: false
 
 steps:
 - task: UseDotNet@2
```

Comment:
for windows, only this task is needed?

Reply:
Yeah, pretty sure the workload comes with the image.